email/welcome-new-xccup.mjml

```diff
@@ -0,0 +1,85 @@
+<!-- Hint: Use this => https://github.com/mjmlio/vscode-mjml -->
+
+<mjml>
+  <mj-body>
+    <mj-section background-color="#08556d">
+      <mj-column>
+        <mj-image width="150px" src="https://next.xccup.net/assets/xccup_white.352249d0.svg"></mj-image>
+      </mj-column>
+    </mj-section>
+    <mj-raw>
+
+      <!-- Intro text -->
+    </mj-raw>
+    <mj-section background-color="#fafafa">
+      <mj-column>
+        <mj-text font-size="20px" font-family="Helvetica Neue" color="#626262">Der neue XCCup</mj-text>
+        <mj-text color="#525252">
+          Liebe XCCup Pilot*innen
+          <p />
+
+          Im letzten Jahr hatte Wolf bereits angekündigt das unser XCCup Entwickler Sascha den Posten nach vielen Jahren gerne abgeben würde und ein Nachfolger gesucht wird.<br />
+
+          Diesem Aufruf sind wir (Kai & Stephan) gefolgt.
+          <p />
+          Da die aktuelle Webseite aus einer Zeit stammt in der Smartphones noch kein Thema waren ist die Bedienung auf mobilen Geräten eher umständlich gewesen. Aus diesem Grund haben wir den XCCup mit Fokus auf mobile Geräte von neu auf programmiert.
+          <br />
+          So gut wie alle Funktionen haben wir dabei übernommen und zusätzlich noch einige Neue hinzugefügt.<p />
+          Wir sind optimistisch, dass wir die neue Webseite zum Saisonstart “live” gehen lassen können. Bis dahin möchten wir euch bitten diese schon mal zu testen und uns zu kontaktieren, wenn etwas nicht wie erwartet funktionieren sollte. Am besten mit Screenshot und Angabe zum verwendeten Gerät und Browser.<br />
+          Ihr könnt dabei auch alte Flüge verwenden. Zu Testzwecken gibt es kein 14-Tages-Limit beim hochladen.<br />
+          Feedback bitte an $$EMAIL EINFÜGEN$$
+          <p />
+
+          Ihr findet die neue Version hier: <a href="https://next.xccup.net" target="_blank">https://next.xccup.net</a>
+          <p />
+          Zum login verwendet bitte eure E-Mail Adresse & euer aktuelles Passwort. Alle Daten werden nach der Testphase gelöscht und haben keine Auswirkungen auf die Wertung 2022. Probiert also gerne alles Mögliche aus.
```

Comment:
Login wird großgeschrieben

Passwörter habe ich logischerweise nicht aus der Altanwendung importiert. Ich habe für jeden Account ein kryptisches Passwort erzeugt.

D.h. die User müssen erstmal auf "Passwort vergessen" klicken.

Reply:
Warum nicht übernehmen die Passwörter?

Reply:
Passwörter lagen nicht im klartext vor sondern wurden auch über irgendeinen Algorithmus gehashed.

Reply:
und das können wir nicht irgendwie umhashen auf unseren? wäre halt geilere UX;)

Reply:
Tell me how... Und dann hättest du noch 800 Passwörter von irgendwelchen Leuten in Klarform vorliegen.

Reply:
warum nicht die alten desalten und hashen und dann mit unserem hash/salt weiterverarbeiten?